authcontroller.py

```diff
@@ -421,7 +421,7 @@ def profile():
 def invite_user():
     """Invite a researcher to join the hub."""
     # For now on boarding of researcher is not supported
-    return "Work in Progress!!!"
+    return "Work in Progress!"
```

Comment:
and that would do the job for the rest of the upcoming things to come, e.g., orcid login